### PR TITLE
Add a HISTORY entry for #132

### PR DESCRIPTION
**Pull request by hroncok**
cc @cjauvin

### CODE REVIEW

File: HISTORY.rst [hunk 1]
```diff
@@ -1,6 +1,15 @@
 History
 =======
 
+Unreleased
+----------
+
+- Cassette names sanitation is now more robust and replaces more special chars;
```
Comment (sigmavirus24) [inline HISTORY.rst]: Let's reword this as:

```
Previous attempts to sanitize cassette names were incomplete. Sanitization has become more thorough which could have some affects on existing cassette files. **This may cause new cassettes to be generated.**
```

Comment (hroncok) [reply HISTORY.rst]: changed. amended.

File: HISTORY.rst [hunk 2]
```diff
@@ -1,6 +1,14 @@
 History
 =======
 
+Unreleased
```
Comment (sigmavirus24) [inline HISTORY.rst]: I usually try to plan out the next release. In this case it's starting out as a bug fix so I'd do:

```
0.8.1 - 2017-xx-yy
------------------
```

This also means less fiddling with the header to make it valid RST and such.
